concourse/tasks/bosh_cancel_all_tasks.yml

```diff
@@ -0,0 +1,15 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/bosh-cli-v2
```

Comment:
we need to use a specific image to avoid unmanaged changes (see concourse/tasks/bosh_delete_apply/task.yml). We are also moving all task definition into it's own directory and name it `task.yml`. Could you please rename it ?